src/ape/managers/chain.py

```diff
@@ -427,14 +424,5 @@ def restore(self, snapshot_id: Optional[SnapshotID] = None):
             snapshot_index = self._snapshots.index(snapshot_id)
             self._snapshots = self._snapshots[:snapshot_index]
 
-        provider.revert(snapshot_id)
+        self.provider.revert(snapshot_id)
         self.account_history.revert_to_block(self.blocks.height)
-
-    def _get_test_provider(self, method_name: str) -> TestProviderAPI:
```

Comment:
Now, we are able to delete this method and no longer have to do stuff like ` self._get_test_provider(TestProviderAPI.revert.__name__)` because it happens at the provider level via `__getattr__`